care/facility/api/viewsets/mixins/history.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.decorators import action
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.decorators'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5345345)